privacyidea/lib/utils/export.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+log = logging.getLogger(__name__)
+
+EXPORT_FUNCTIONS = {}
+IMPORT_FUNCTIONS = {}
+
+
+def register_export(name=None):
```

Comment:
We are still missing a documentation like a docstring for the exporter and import, so that a developer knows how this is to be used.

"""
The register_export decorator is supposed to decorate a function that exports the configuration data from a given module like resolvers, realms, events...

The exporter function simply needs to return a dictionary contained the data.

@register_export
def export_events():
     .... some code ...
"""

And a docstring, so that a developer knows, how to use the import decorator.

"""
The decorator register_import is used to register an importer function, that can import data to a privacyidea section,
that was previously exported by the exporter.

Usually this is a dictionary. The exporter and importer of the section take care of handling the data correctly.

The priority can be used to define, which section are imported before other sections. Like resolvers need to be imported before realms, since realms depend on the resolvers.

example:

@register_import(prio=1)
def import_events():
 """

Reply:
Done in f6f28b91